### ISSUE TITLE
Launch option to get only gdb output in console

### ISSUE THREAD

**Issue by Groogy** [WebFreak001/code-debug#451]
Hi! I have gdb setup to print out the info I want, and basically just want a fullscreen console that gets its breakpoints set from vscode. I can't use the external terminal as described in issue #76. 

Was wondering if it would be possible to ask for an option in the launch config where you set the extension to only provide the gdb output? Would this be something hard to accomplish? Preferably I'd like to keep working in the integrated console.

Example of what I have and what I want
![image](https://github.com/user-attachments/assets/610f832f-409e-4bb2-97f3-7c5467f03e0a)
![image](https://github.com/user-attachments/assets/b9e71b6b-b717-439f-a664-6775e71a130a)

**Comment by GitMensch** [comment]
If this is added then GDB would need to be started in either the integrated or an external terminal - but within a shell, as you have a bunch of terminal escape sequences in here.

I _think_ that would be possible as you can use a similar setup in Vim's Termdebug, which also integrates debugging via mi, as is done in this extension. Effectively this would mean 1 application terminal (either integrated or external), 1 debugging console as it is now and 1 gdb tty in another integrated/external terminal.

**Comment by Groogy** [comment]
Do you mean it would pop up as an integrated terminal, like a task? I'd be fine with that. 

My use case is that I want the nice functionality of starting the gdb session the extension provides, but then I want to mainly work with the gdb session through a terminal.

**Comment by GitMensch** [comment]
That's a very rare use-case, because the extension's purpose is about debugging _in the IDE_.

Before suggesting a solution path...
What do you mean with "**mainly** work ... in the terminal"? What would you like to be kept to be done in the extension, other than the launch?

**Comment by Groogy** [comment]
Yeah I can agree it's a very specific use-case and might be out of scope for the extension.

What I want in essence is to keep the nice to have to "setup the gdb session" while working in VSCode, specifically setting breakpoints. Right now I have a file I put in my starting commands to gdb, but it would be nicer to just be able to click what line I want to break at. But once I am actually debugging I just prefer the CLI interface a lot more.

I guess I want the best of both worlds 😅

**Comment by GitMensch** [comment]
closed as off-topic for the GDB extension, as it is about handling GDB communication via MI to implement the debug adapter protocol, not to provide "another way to start gdb"
